models/ht_neuron.cpp

```diff
@@ -208,11 +224,13 @@ nest::ht_neuron::Parameters_::Parameters_()
   , AMPA_Tau_2( 2.4 ) // ms
   , AMPA_E_rev( 0.0 ) // mV
   , NMDA_g_peak( 0.075 )
-  , NMDA_Tau_1( 4.0 )  // ms
-  , NMDA_Tau_2( 40.0 ) // ms
-  , NMDA_E_rev( 0.0 )  // mV
-  , NMDA_Vact( -58.0 ) // mV
-  , NMDA_Sact( 2.5 )   // mV
+  , NMDA_Tau_1( 4.0 )        // ms
+  , NMDA_Tau_2( 40.0 )       // ms
```

Comment:
If you think it's worth it (I'm not sure), having all ``tau``s in lowercase would be more coherent, though it would break existing codes... and ``Tau_m`` instead of ``tau_m`` is really too bad since it's supposed to be common to many models...